A few bits of back and suggestions (#1)
ebuchman/blog: content/posts/bitcoin.md

Tesla, a bastion of our clean energy future, is helping us transition
from fossil fuels to electricity.
Bitcoin, a speculative digital asset, is using more energy 

> than Switzerland and is (or at least was!) largely powered by dirty coal mined in China. 

Author: Unless you plan to address cynophobia, I'd avoid that phrasing.

monopolistic, American consumerism is not particularly nurturing for a sustainable future. 
Tesla is perhaps American capitalism's best foot forward.

> Two is the supply chain of modern technology.

Author: Suggested these just cause the sequence `One, Two, Third` struck me as off. An alternative would be `First, Second, Third` or `One, Two, Three`.

The mining industry, for metals like Lithium, Coltan, and more,
is [riddled with environmental and human rights abuses][supply-chain].
These are not so much measured in carbon emissions (though there are plenty)
as in pollution and destruction of natural habitats and social relationships.
And a car battery requires many orders of magnitude more Lithium than your Macbook. 
Clean energy has a very dirty supply chain, and little incentive to improve it.

A third is cars themselves. The complete centrality of cars in the
organization of urban life is one of the great tragedies
of our time. Sustainability is intimately bound up with how we organize ourselves
on the surface of the planet. Cars, electric or otherwise, are not
particularly conducive to the more human-scale and bio-diverse patterns that are 
critical to sustainability. To the extent that we have cars, they should probably be electric. 
But more than anything, we need less cars. Denmark has the right idea. As does
anyone who's ever gotten a street closed to cars, for any duration. Respect.

Now I'm not trying to diss Tesla, per se. 
Tesla is the beacon of a research and development process that is itself critical to

> sustainability. <!-- I don't follow the reasoning of the next sentence: ---> And if we want to make progress in batteries, which we should, cars are a great

Reviewer: ```suggestion
sustainability. And if we want to make progress in batteries, which we should, cars are a great
```

> way to go. But the fundamental problems we're facing today don't seem so much technical as they

Author: That is: why are cars a good way to make progress on batteries? Seems like those are orthogonal.

Reviewer: ah, just that they provide a good socioeconomic impetus for battery. car gas tanks are one of the most dominant energy stores in peoples experience so from a product-market perspective, cars are probably a good place battery innovation to get funded and delivered

Author: That makes sense. So, iiuc, the reasoning is something like: given the current way we motivate and incentivize research, increasing the demand for, and so the profits to be gained from, electronic cars, may be the best bet to getting research directed to battery technology? I see how this could parallel the argument about bitcoin demand driving green tech innovation.

> way to go. But the fundamental problems we're facing today don't seem so much technical as they

Reviewer: ```suggestion
vehicle to ground that R&D. But the fundamental problems we're facing today don't seem so much technical as they
```

But Bitcoin isn't a precious metal. And it's not something produced by a company, nor a government, 
nor really any distinctly identifiable organization.

> Bitcoin is an open-source, verifiable, transparent, decentralized, 

Author: iiuc, companies and governments are also collectives, and since you use the qualified "collective" in the next sentence...

Reviewer: ```suggestion
Bitcoin is a collective. An open-source, verifiable, transparent, decentralized, 
```

Sure but I'm trying to use it here more for its amorphous/illegible qualities as distinct from well bounded/definable "organizations"

At least some of this appears due to a collapse in the literal values of money as a
representative media. Unmoored from any real world process, 

> <!-- I like alliteration, but this is a lot of it and I tripped up a bit this clause :) -->

Reviewer: Hah thanks, I can't help my self sometimes, gonna let this one lie.
```suggestion
```

as a highly liquid (though volatile) reserve asset on corporate balance sheets.
It's also finding increasing utility among the people of countries with unstable currencies and authoritarian governments.

<!-- The latter seems to really undercut the reality. The vast majority of bitcoin use has been for this speculative, finance, no? -->
Of course it's also being used increasingly in ransomware payments and as fodder
for speculative finance. In ransomwhere, it tragically illuminates the pervasive insecurity 
and fragility of the systems underlying our critical infrastructure. In finance, it fuels the same kind
of [leveraged and nebulous behaviour][tether] as the rest of the [shadow banking] system.
Perhaps the greatest risk to Bitcoin is that it is [swallowed up by Wall
St][epsilon-theory-bitcoin].

> It may be necessary, but it's certainly not sufficient.

Reviewer: ```suggestion
But Bitcoin is not alone the solution to the world's monetary problems. It may be necessary, but it's certainly not sufficient.
```

Proof-of-Work entails a specific kind of cryptographic computation resembling
a numerical puzzle that is difficult to solve but easy to verify. If
computing power were evenly distributed throughout humanity (which of course

> <!-- why "could never be"? -->

Reviewer: ```suggestion
```

> it's not and could never be), Bitcoin's Proof-of-Work mining would be a sort of Universal Basic Job, 

Author: I think "it's not" probably suffices, and you don't lose people who think it is possible to have an equitable distribution of resources.

> it's not and could never be), Bitcoin's Proof-of-Work mining would be a sort of Universal Basic Job, 

Reviewer: ```suggestion
it's not), Bitcoin's Proof-of-Work mining would be a sort of Universal Basic Job, 
```

destruction and salvation of the human species. We have only to ask ourselves,
"Is the game worth the candle?". And in this sense, the world faces much bigger
problems than the energy usage and e-waste of Bitcoin. I think we'd do better to face those problems with

> Bitcoin at our back than opposing it.

Reviewer: ```suggestion
Bitcoin at our back than opposing it.
```
```suggestion
Bitcoin at our back than without.
```

Many people have heard the phrase "blockchains, not Bitcoin", 
meant to express that blockchain technology is a meaningful innovation,

> but Bitcoin can be largely ignored. Of course, I don't believe Bitcoin can or

Author: Just cause I didn't generally see third person used in this case of case early in the piece.

the cryptographically verifiable energy expenditure in a remote data center that
is today's Proof-of-Work, to the communally verifiable delivery of life sustaining care work that is tomorrow's Proof-of-Care and [Proof-of-Plant][proof-of-plant]. 
Proof-of-Stake may be the next step in that journey - somewhat more subjective, less anonymous, more socially aware than Proof-of-Work. 

> <!-- PoW has the same problems, right? With more bitcoin you can own more computer, which means you can amass and cointrol more bitcoin? -->

Author: Or else you could maybe give a line pointing indicating the different level at which you're identifying this tendency in the case of PoS vs. PoW.
